### PR TITLE
Removed market iteration

### PR DESCRIPTION
**Pull request by declansullivan**
Changed how markets are pulled from the database slightly.  UUID has not been implemented yet, as that will be a pretty large change to a good chunk of the code, so this is more of a quick fix.  I fixed the key of the one market that was causing issues, and made each key based on the market name and address, with offending characters stripped.  This should be better than what we had before, and it allows for immediate access of information, instead of looping through markets.

### CODE REVIEW

File: routes/markets.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ router.get('/', function(req, res, next) {
 				address: childData.address, 
 				size: childData.storeType, 
 				zip: childData.zip,
-				level: childData.marketLevel
+				level: childData.marketLevel,
+				key: childData.marketName.replace(/[^0-9a-zA-Z," ]/gi, '') + ", " + childData.address.replace(/[^0-9a-zA-Z," ]/gi, '')
```
Comment (sumeet-bansal) [inline routes/markets.js]: maybe extract this into its own function, e.g. `generateDocumentKey`

File: routes/marketdata.js [hunk 2]
```diff
@@ -13,48 +13,42 @@ const ref = db.ref("live_weller");
...
-            let questions = []
-            let missed = [];
-
-            if (childData.marketInfo.marketName === marketName) {
-                status = "Questions to fix to get to level " + (parseInt(childData.marketInfo.marketLevel) + 1);
+router.get('/:marketKey', function(req, res, next) {
+    const market = marketsRef.child(req.params.marketKey);
+
+    market.once('value', function(snapshot) {
+        let questions = []
```
Comment (sumeet-bansal) [inline routes/marketdata.js]: should move these down to their respective loops (right before the first time they're used) for clarity

File: routes/markets.js (hunk 1 above)
Comment (declansullivan) [reply routes/markets.js]: Will do

**Comment by declansullivan** [comment]
One more quick change for consistency with how I generate the keys, sorry!

**Pull request merged**